### ISSUE TITLE
Very cool implementation.

### ISSUE THREAD

**Issue by RyanBurnside** [flambard/tagbody#4]
This was a great little project. 
Excellent work, I think there is a swath of Clojure programmers who never really dig into the language more than using it like it was Python. It's refreshing to see somebody who's embraced it as a dialect of Lisp. Unstructured programming is generally not a good thing but this is more about what you CAN do with a carefully placed DSL.

**Comment by gwangjinkim** [comment]
Indeed! I was just looking at Common Lisp's Tagbody and stumbled over this. It is amazing!
